### ISSUE TITLE
Is it possible that the agent asks the question in case of form filling activity so that the user knows what what information they need to provide.

### ISSUE THREAD

**Issue by Shiv270895** [lavague-ai/LaVague#445]
**Is your feature request related to a problem? Please describe.**
I am trying to fill a form where I dont know the attributes upfront so I was thinking if the agent itself asks the question about the attribute and I can provide all the necessary values accordingly.

**Describe the solution you'd like**
The solution which i would like a demo solution of my issue or a link where i can find something similar which is done through La-vague agents.

**Describe alternatives you've considered**
I haven't considered any alternatives yet as i couldn't find anything relatable on the documentation.

**Additional context**
There is no additional context for this, Just hoping for a resolution and response.

**Comment by dhuynh95** [comment]
If it's ok can you provide URL + objective you want the agent to do?

**Comment by paulpalmieri** [comment]
@Shiv270895 do you basically want the agent to be able to do a back and forth with you when it's missing info to complete a given step ?

**Comment by Shiv270895** [comment]
Hello @dhuynh95 ,
We can take the same form which is used in the documentation purpose for making us understand how LaVague actually works. Its just that instead of user being intelligent and providing the values, The agent should ask the user for the values from the user this is my objective. So that it becomes easier for user also when they don't know about the attributes.

**Comment by Shiv270895** [comment]
Hello @paulpalmieri ,
Yes I want the agent to do the back and forth communication to the user so that its easier for them.

**Comment by Shiv270895** [comment]
Hello @dhuynh95 ,
We can take this form for reference
URL - https://form.jotform.com/241572484030451
Objective - I want that the agent should ask about the attribute's value from the user instead of user giving everything in the Instructions. Let's suppose here we have only 4 attributes but in other cases we can have 15-20 attributes so in that case it will be exhausting and tiring for user to go and write everything in the Instruction in that case i think the user would prefer directly filling out the form. So is it possible that the agent becomes intelligent and asks for value's of attributes.
If this can be achieved through LaVague then it would be really helpful.

So if we have something like this in LaVague then Please help me with the documentation for achieving this.

Thanks & Regards,
Shiv kumar

**Comment by signalprime** [comment]
Or to cover more use cases, there could be an option that tells the agent to query the human when it doesn't have the data requested on a form.  I came here looking for conversations about getting the agent to ask the human for help when it needs guidance.

**Comment by lyie28** [comment]
Yeah, it's a nice feature idea @Shiv270895 . @adeprez, what do you think? Is this something we will cover, or perhaps a `help-wanted` issue?

**Comment by Shiv270895** [comment]
Hi @lyie28 , 
Hope so we can see this new feature soon in LaVague.

**Comment by Shiv270895** [comment]
Hi @lyie28 ,

Hope you are doing well.
Do we have this feature now, or till when we can get this feature for using/testing.

Regards,
Shiv Kumar

**Comment by lyie28** [comment]
Hi @Shiv270895, we don't have this feature yet. 

Internally, we have some other priorities around performance that we are focusing on first.

We did have some interest from the community from contributors and started a chat here: https://discord.com/channels/1216089456134586388/1265250774913388605 - but it has all gone quiet for now so I am not sure that anyone is actively working on this right now.

**Comment by Shiv270895** [comment]
Hi @lyie28 ,

Thankyou for the response, like till when we can expect this feature to be out, like are you guys planning for this feature?

Regards,
Shiv Kumar
